### ISSUE TITLE
Oven Fire and Hot Tub Fire- Unlimited

### ISSUE THREAD

**Issue by Thump1t** [Grantapher/ValheimPlus#62]
The oven obviously has a flame/fire source, but it does not have an option to set to unlimited flame like the campfire/torches/etc.

It would be very nice if the Oven could have the same settings as the campfire/torches, so once at max fuel, it stays there and always runs without using any wood, or pull wood from storage, etc. 

Same with the hot tub, it has a fuel source, but you always have to put wood in it to run.

**Comment by CaptainOblivious04** [comment]
This would be a great feature to add. My end game bases usually are heated by ovens. If they cannot be set to infinite fuel maybe add a max limit like the kilns and smelters

**Comment by snorkyware** [comment]
Got the oven working, still can't even find any code relating to the hot tub.

**Comment by CaptainOblivious04** [comment]
> Got the oven working, still can't even find any code relating to the hot tub.

How long does it usually take to upload the changes to Thunderstore?

**Comment by snorkyware** [comment]
> How long does it usually take to upload the changes to Thunderstore?

Not sure. I'm just submitting code updates.

I think I've found the hot tub, it thinks it's a smelter.

**Comment by CaptainOblivious04** [comment]
> > How long does it usually take to upload the changes to Thunderstore?
> 
> Not sure. I'm just submitting code updates.
> 
> I think I've found the hot tub, it thinks it's a smelter.

Alright. I have no clue how the code works for the game. Really cool how you guys are doing this! Cheers!

**Comment by Thump1t** [comment]
> Release here for manual install: https://github.com/snorkyware/ValheimPlus/releases/tag/0.9.13-snorkyware See bottom of: https://github.com/snorkyware/ValheimPlus/blob/grantapher-development/valheim_plus.cfg for new config options.

This config update has the oven and hot tub options in it?

**Comment by snorkyware** [comment]
> This config update has the oven and hot tub options in it?

Yes. Oven, hot tub, sap collector and shield generator.

**Comment by CaptainOblivious04** [comment]
> > Release here for manual install: https://github.com/snorkyware/ValheimPlus/releases/tag/0.9.13-snorkyware See bottom of: https://github.com/snorkyware/ValheimPlus/blob/grantapher-development/valheim_plus.cfg for new config options.
> 
> This config update has the oven and hot tub options in it?

How do I manually install this to ThunderStore Mod Manager? Manually update V+ there.

**Comment by CaptainOblivious04** [comment]
> > Release here for manual install: https://github.com/snorkyware/ValheimPlus/releases/tag/0.9.13-snorkyware See bottom of: https://github.com/snorkyware/ValheimPlus/blob/grantapher-development/valheim_plus.cfg for new config options.
> 
> This config update has the oven and hot tub options in it?

Instead of infinite fuel is there an option to increase its max fuel? Like how the smelters can be raised from 20 to 100? The oven can go from 10 as its max to 50 ish. Same with the shield generator. Is it possible to implement it? Thanks

**Comment by CaptainOblivious04** [comment]
> > This config update has the oven and hot tub options in it?
> 
> Yes. Oven, hot tub, sap collector and shield generator.

Instead of infinite fuel is there an option to increase its max fuel? Like how the smelters can be raised from 20 to 100? The oven can go from 10 as its max to 50 ish. Same with the shield generator. Is it possible to implement it? Thanks

**Comment by snorkyware** [comment]
> Instead of infinite fuel is there an option to increase its max fuel? Like how the smelters can be raised from 20 to 100? The oven can go from 10 as its max to 50 ish. Same with the shield generator. Is it possible to implement it? Thanks

It certainly is possible. I'll have a look at it if I get a chance.

**Comment by Thump1t** [comment]
@snorkyware I d/l the new .dll and put it in my game files as I do with the V+ updates, but I didn't see the new config options in the V+.config file. Am I doing something wrong?

**Comment by snorkyware** [comment]
> @snorkyware I d/l the new .dll and put it in my game files as I do with the V+ updates, but I didn't see the new config options in the V+.config file. Am I doing something wrong?

Yes, V+ is old-fashioned by modern Valheim plugin standards, it doesn't use BepInEx configs which I think is the behaviour you're expecting. Add the new config options manually, referring to default config file:

> Release here for manual install: https://github.com/snorkyware/ValheimPlus/releases/tag/0.9.13-snorkyware See bottom of: https://github.com/snorkyware/ValheimPlus/blob/grantapher-development/valheim_plus.cfg for new config options.

**Comment by snorkyware** [comment]
> @snorkyware. I'm terribly blind, I've went above and clicked on the config file you said the new addons were listed at the bottom of, and I just can't seem to find the code to copy/paste into mine like you said to do in the last post. I hate to ask, but can you link the exact file I need to look in to find the new code I need to paste into my config? For the life of me I just can't see it.

That's because I shuffled all the branches around, trying to make acceptable pull requests. Sorry about that.
See the main valheim_plus.cfg now: https://github.com/Grantapher/ValheimPlus/blob/grantapher-development/valheim_plus.cfg
